### DISCUSSION TITLE
FSearch open folder opened in VS Code, not in Thunar

### DISCUSSION THREAD

**Discussion by benyaminl** [cboxdoerfer/fsearch#437] [Q&A] [answered]
Hello, Here I am again, I'm a Fedora XFCE user, tbs I'm on Fedora 36, using fsearch rpm, my question is how to change default file manager in fsearch ? In default Mime config, it's set to thunar, but seems fsearch think otherwise and use vs code. Is it a bug, or intended feature? Thank you 

Screenshoot
![image](https://user-images.githubusercontent.com/2817332/197910134-f3ba42cc-5dc3-4aa7-8e5b-ab44bc4df2f6.png)

**Comment by cboxdoerfer** [comment, marked as answer]
What is the output of `xdg-mime query default inode/directory` ?

**Reply by benyaminl** [reply]
Ah... what a messs... 
![image](https://user-images.githubusercontent.com/2817332/197951692-fa78e012-8a1e-4ded-8010-321b555cbb37.png)
Seems It doesn't value at first in fedora 36, and just added after installing code. Change to Thunar, fix the issue. 

Thanks for clarifying.
